viewer/packages/cad-model/src/Cognite3DModel.ts

```diff
@@ -284,7 +284,8 @@ export class Cognite3DModel extends THREE.Object3D implements CogniteModelBase,
    * Cleans up used resources.
    */
   dispose(): void {
-    this.children = [];
+    this.cadNode.dispose();
+    this.clear();
```

Comment:
I was testing it and it seems that potree manages garbage collection inside itself. Right when the model is removed the memory is not freed but when another model is loaded it cleans everything from the last one. If we want to have cleaned model data right after removal, I can look into it in separate PR.

Reply:
Ok, tested it again and it seems that point cloud models have exactly the same issue with memory leaks because right now we are not disposing of anything plus `removePointCloud` method of `PotreeGroupWraper` is not implemented properly as it is not removing a few references of models inside itself. So this all things should be in another PR which I will create at some point because right now I think it is better to focus on other stuff.